pkg/asset/cluster/tfvars.go

```diff
@@ -36,24 +40,76 @@ func (t *TerraformVariables) Dependencies() []asset.Asset {
 		&installconfig.InstallConfig{},
 		&bootstrap.Bootstrap{},
 		&machine.Master{},
+		&machines.Master{},
 	}
 }
 
 // Generate generates the terraform.tfvars file.
 func (t *TerraformVariables) Generate(parents asset.Parents) error {
 	installConfig := &installconfig.InstallConfig{}
-	bootstrap := &bootstrap.Bootstrap{}
-	master := &machine.Master{}
-	parents.Get(installConfig, bootstrap, master)
+	bootstrapIgnAsset := &bootstrap.Bootstrap{}
+	masterIgnAsset := &machine.Master{}
+	mastersAsset := &machines.Master{}
+	parents.Get(installConfig, bootstrapIgnAsset, masterIgnAsset, mastersAsset)
 
-	bootstrapIgn := string(bootstrap.Files()[0].Data)
+	bootstrapIgn := string(bootstrapIgnAsset.Files()[0].Data)
 
-	masterIgn := string(master.Files()[0].Data)
+	masterIgn := string(masterIgnAsset.Files()[0].Data)
 
-	data, err := tfvars.TFVars(installConfig.Config, bootstrapIgn, masterIgn)
+	var mastersList metav1.List
+	err := yaml.Unmarshal(mastersAsset.MachinesRaw, &mastersList)
+	if err != nil {
+		return errors.Wrap(err, "unmarshal master machine list")
+	}
+
+	masters := make([]*clusterapi.Machine, 0, len(mastersList.Items))
+	for _, item := range mastersList.Items {
+		var master clusterapi.Machine
+		err = yaml.Unmarshal(item.Raw, &master)
+		if err != nil {
+			return errors.Wrap(err, "unmarshal master machine item")
+		}
+		masters = append(masters, &master)
+	}
+
+	var vpcCIDR string
+	if installConfig.Config.Platform.AWS != nil {
+		vpcCIDR = installConfig.Config.Platform.AWS.VPCCIDRBlock
+	}
+
+	var bridge string
+	var image string
+	if installConfig.Config.Platform.Libvirt != nil {
+		vpcCIDR = installConfig.Config.Platform.Libvirt.Network.IPRange
+		bridge = installConfig.Config.Platform.Libvirt.Network.IfName
+		if installConfig.Config.Platform.Libvirt.DefaultMachinePlatform == nil {
```

Comment:
Hmm, we should do something like we do in Master Asset. this is not sustainable.

Reply:
> Hmm, we should do something like we do in Master Asset. this is not sustainable.

Extracting the image here?  Our Terraform handling is currently broken by assuming a single libvirt image, while the Go types allow for per-machine(-set) images.  Eventually we're going to either want to remove the flexibility from our Go (if we want to go down the "everyone will use RHCOS and won't care about the details" path) or add it to our Terraform (if we want want to go down the "folks may want to choose on a per-machine(-set) basis" path).  I'm fine punting on that for this PR, but let me know if you have clarity on the long-term direction (or if you think it needs to be handled here).